app/src/main/java/org/fossasia/openevent/general/settings/SettingsFragment.kt

```diff
@@ -106,4 +116,19 @@ class SettingsFragment : PreferenceFragmentCompat(), PreferenceChangeListener {
         val alert = builder.create()
         alert.show()
     }
+
+    override fun onResume() {
+        val activity =  activity as? AppCompatActivity
+        activity?.supportActionBar?.setDisplayHomeAsUpEnabled(true)
+        activity?.supportActionBar?.title = "Settings"
+        super.onResume()
+    }
+
+    override fun onDestroy() {
+        val activity =  activity as? AppCompatActivity
+        activity?.supportActionBar?.setDisplayHomeAsUpEnabled(false)
+        activity?.supportActionBar?.title = "Profile"
```

Comment:
Same problem as previously

Reply:
Yeah, this a problem actually. onResume method is not working in Profile Fragment.

Reply:
Check why onResume isn't called

Reply:
Actually onResume works as it should. Whenever the app is on hold and again opened, the title is updated to Profile, but when we click the back button of SettingsFragment, it doesn't work peculiarly

Reply:
onResume should still be called

Reply:
I created a new project and tried, it is working flawlessly

Reply:
Was the same code utilized like are you just setting the title in the onResume method or anything else is being done ?

Reply:
I'm setting the title in onResume

Reply:
Backpress is also working? Like in my case the only problem was that on backpress of Settings Fragment , The profile fragment title would change to Settings and remain as that!

Reply:
I will try again then :+1:

Reply:
I pushed the code, onBackPress of Settings fragment it doesn't work still.